invenio_rdm_records/records/processors/tiles.py

```diff
@@ -23,23 +23,27 @@ class TilesProcessor(RecordFilesProcessor):
 
     @property
     def valid_exts(self) -> list:
-        """Return valid extenstions for tiles generation from config/default."""
+        """Return valid extensions for tiles generation from config/default."""
         return current_app.config.get(
             "IIIF_TILES_VALID_EXTENSIONS", ["tiff", "pdf", "jpeg", "png"]
         )
 
     def _can_process(self, draft, record) -> bool:
         """Checks to determine if to process the record."""
-        return (
-            # Check if tiles generation is enabled...
-            current_app.config.get("IIIF_TILES_GENERATION_ENABLED", False)
-            and (
-                # ...has convertable files (i.e. images)
-                bool(set(self.valid_exts).intersection(record.files.exts))
-                # ...or has already converted files
-                or (record.media_files.enabled and "ptif" in record.media_files.exts)
-            )
-        )
+        for file_type in ['media_files','files']:
+            files = getattr(record, file_type, {})
```

Comment:
```suggestion
            files = getattr(record, file_type)
```

We don't need a default value, as `.exts` will fail later causing an confusing error.